pojo/src/test/java/com/spotify/hamcrest/pojo/IsPojoTest.java

```diff
@@ -112,18 +112,39 @@ public void testMismatchFormatting() throws Exception {
 
     assertThat(description.toString(), is(
         "SomeClass {\n"
-        + "  ...\n"
         + "  baz(): SomeClass {\n"
-        + "    ...\n"
         + "    foo(): was <42>\n"
-        + "    ...\n"
```

Comment:
just to double check, these ellipses are removed because `baz` only has one field and so they never really made any sense?